easybuild/easyconfigs/g/git-annex/git-annex-10.20230802-GCCcore-12.2.0.eb

```
@@ -0,0 +1,45 @@
+easyblock = 'MakeCp'
+
+name = 'git-annex'
+version = '10.20230802'
+
+homepage = 'https://git-annex.branchable.com'
+description = """git-annex allows managing large files with git, without storing the file contents in git. It can sync,
+backup, and archive your data, offline and online. Checksums and encryption keep your data safe and secure. Bring the
+power and distributed nature of git to bear on your large files with git-annex."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+
+dependencies = [
+    ('GHC', '9.2.2', '-x86_64', SYSTEM),
+    ('GCC', '12.2.0', '', SYSTEM),
+    ('Stack', '2.11.1', '-x86_64', SYSTEM),
+    ('git', '2.38.1', '-nodocs'),
+]
+
+sources = [{
+    'git_config': {'url': 'git://git-annex.branchable.com',
+                   'repo_name': '%(name)s',
+                   'tag': '%(version)s',
+                   'clone_into': '%(name)s-%(version)s',
+                   },
+    'filename': '%(name)s-%(version)s.tar.gz',
+}]
+
+checksums = ['62e4b4d9ef5ecf587a854a761927b1dbf4d646a0b8570e594ff48e3dac3719b5']
```

sorry, should have spotted this earlier, but with git clones you should not add a checksum, as those are not guaranteed to be the same
```suggestion
checksums = [None]
```

---

Ah okay, I was wonndering a little, but didn't question it. I'll commit it as suggested.